mender-client-acceptance-testing/deb-requirements.txt

```diff
@@ -55,3 +55,5 @@ sysstat
 texinfo
 wget
 zlib1g-dev
+rustc
+cargo
```

Comment:
based on [this note](https://cryptography.io/en/latest/installation.html#rust) I think the way to go would be to rely on the updated wheel to get around installing these two... in any case it would be nice having them inserted in the alphabetical order the file is in - but for a dependency list there are likely better grouping/ ordering options 🤷‍♂️...

Reply:
@mzedel I tried that but it doesn't work in these base images. It works for [backend-tests](https://github.com/mendersoftware/integration/blob/master/backend-tests/docker/Dockerfile#L16) where the base Docker image is `python:3`, for example :man_shrugging:

Reply:
ah ok apparently the `cryptography==3.4.5` package is just not updated yet for `18.04` and seems to install just fine in the `20.04` image... if you think that is a recurring issue with the dependencies, feel free to go ahead and merge... otherwise I guess it's time to change the base image to something that can keep up with the dependabot schedule

Reply:
@mzedel We cannot upgrade to `ubuntu:20.04` as this image is still used to test old yocto branches (which, for instance, still use Python 2).

A possibility would be to build two different images, like "current" and "legacy". But mender-qa pipeline would get very complicated...

Reply:
(and before I merge, I need to solve the other problem wrt image being too big)